packages/next/src/server/app-render/types.ts

```diff
@@ -111,6 +111,7 @@ export type RenderOptsPartial = {
   runtime?: ServerRuntime
   serverComponents?: boolean
   assetPrefix?: string
+  crossOrigin?: '' | 'anonymous' | 'use-credentials' | undefined
```

Comment:
`renderOpt.crossOrigin` exists on `PagesRenderOptsPartial` but it is missing on `AppRenderOptsPartial` here.